### PR TITLE
Don't log exception if Linux specific file does not exist

### PR DESCRIPTION
**Pull request by bjorncs**

### CODE REVIEW

File: container-disc/src/main/java/com/yahoo/container/jdisc/metric/MetricUpdater.java [hunk 1]
```diff
@@ -80,6 +80,9 @@ private long count_mappings() {
             long count = 0;
             try {
                 Path p = Paths.get("/proc/self/maps");
+                if (!p.toFile().exists()) { // E.g. MacOS
```
Comment (gjoranv) [inline container-disc/src/main/java/com/yahoo/container/jdisc/metric/MetricUpdater.java]: Please convert to one-liner

File: container-disc/src/main/java/com/yahoo/container/jdisc/metric/MetricUpdater.java [hunk 2]
```diff
@@ -97,6 +100,9 @@ private long count_open_files() {
             long count = 0;
             try {
                 Path p = Paths.get("/proc/self/fd");
+                if (!p.toFile().exists()) { // E.g. MacOS
```
Comment (gjoranv) [inline container-disc/src/main/java/com/yahoo/container/jdisc/metric/MetricUpdater.java]: Please convert to one-liner

**Review by gjoranv — CHANGES_REQUESTED** [review]

**Comment by bjorncs** [comment]
@gjoranv PTAL

**Comment by gjoranv** [comment]
Looking great!

**Review by gjoranv — APPROVED** [review]

**Pull request merged**